src/WorkflowManager/ConditionsResolver/Resovler/Conditional.cs

```diff
@@ -273,12 +273,12 @@ public bool Evaluate(IFormatProvider? culture = null)
                 NOT_CONTAINS => !ContainsEvaluate(),
                 EQUAL_EQUALS => string.Equals(LeftParameter.Trim(), RightParameter.Trim(), StringComparison.InvariantCultureIgnoreCase),
                 NOT_EQUAL => !string.Equals(LeftParameter.Trim(), RightParameter.Trim(), StringComparison.InvariantCultureIgnoreCase),
-                GTstr => Convert.ToInt16(LeftParameter, culture) > Convert.ToInt16(RightParameter, culture),
-                LTstr => Convert.ToInt16(LeftParameter, culture) < Convert.ToInt16(RightParameter, culture),
-                GE => Convert.ToInt16(LeftParameter, culture) >= Convert.ToInt16(RightParameter, culture),
-                EG => Convert.ToInt16(LeftParameter, culture) >= Convert.ToInt16(RightParameter, culture),
-                LE => Convert.ToInt16(LeftParameter, culture) <= Convert.ToInt16(RightParameter, culture),
-                EL => Convert.ToInt16(LeftParameter, culture) <= Convert.ToInt16(RightParameter, culture),
+                GTstr => Convert.ToDecimal(LeftParameter, culture) > Convert.ToDecimal(RightParameter, culture),
```

Comment:
Could you write a quick test to cover larger integers being parsed correctly please? Would help with this not regressing in the fuuture.